python/cudf/io/csv.py

```diff
@@ -56,6 +56,9 @@ def read_csv(filepath, lineterminator='\n',
         Number of rows to be skipped from the start of file.
     skipfooter : int, default 0
         Number of rows to be skipped at the bottom of file.
+    compression : {'infer', 'gzip', 'zip', None}, default 'infer'
+        For on-the-fly decompression of on-disk data. If 'infer', then detect
```

Comment:
```suggestion
compression : {'infer', 'gzip', 'zip', None}, default 'infer'
    For on-the-fly decompression of on-disk data. If ‘infer’, then detect compression from the following extensions: ‘.gz’,‘.zip’ (otherwise no decompression). If using ‘zip’, the ZIP file must contain only one data file to be read in. Set to None for no decompression.
```

Do zip files have the same one-file requirement?

Reply:
Yes. GZIP are inherently single-file, while for ZIP we'll pick only the first non-zero-sized file, this matches Panda's behavior afaik.

Reply:
My suggested change would throw a flake8 failure, but can you add the information to the docstring then mark this review as resolved?